shell/src/main/java/org/apache/accumulo/shell/commands/ScanCommand.java

```diff
@@ -270,6 +271,16 @@ protected ScanInterpreter getInterpreter(final CommandLine cl, final String tabl
 
   protected void fetchColumns(final CommandLine cl, final ScannerBase scanner,
       final ScanInterpreter formatter) throws UnsupportedEncodingException {
+
+    if ((cl.hasOption(scanOptCf.getOpt()) || cl.hasOption(scanOptCq.getOpt()))
+        && cl.hasOption(scanOptColumns.getOpt())) {
+
+      String formattedString =
+          String.format("Options - %s AND (- %s" + " OR - %s are mutually exclusive )",
+              scanOptColumns.getOpt(), scanOptCf.getOpt(), scanOptCq.getOpt());
```

Comment:
```suggestion
          String.format("Option -%s is mutually exclusive with options -%s and -%s.",
              scanOptColumns.getOpt(), scanOptCf.getOpt(), scanOptCq.getOpt());
```
Some suggestions on wording to make it easier to understand and fixed spacing.